include/osquery/core.h

```diff
@@ -103,7 +103,7 @@
 #define OSQUERY_DB_HOME OSQUERY_HOME
 #define OSQUERY_SOCKET OSQUERY_DB_HOME "/"
 #define OSQUERY_LOG_HOME "/var/log/osquery/"
-#define OSQUERY_CERTS_HOME "/usr/share/osquery/certs/"
```

Comment:
If you're going to update this path as well, you must also update `make_linux_package.sh` in addition to `make_osx_package.sh`